### ISSUE TITLE
Complete Life Sciences Reporting Summary

### ISSUE THREAD

**Issue by ahwagner** [ahwagner/vicckb#36]
> As you are already aware, Nature Research is taking an active approach to improving our transparency standards and increasing the reproducibility of all of our published results. Detailed information on experimental design and reagents that was formerly required to be included in the body of your paper is now collected on our recently updated Life Sciences Reporting Summary, which will be published alongside your paper. Please provide an updated version of the Reporting Summary with your final files.

**Comment by ahwagner** [comment]
[nr-reporting-summary.pdf](https://github.com/ahwagner/vicckb/files/4020535/nr-reporting-summary.pdf)

**Comment by ahwagner** [comment]
In response to code availability comments from EPC_checked and RS_checked, we have added a “code availability” section and restructured other methods sections previously describing code availability as subheaders in this section. The comments from RS_checked under “Life sciences study design” appear to be the default descriptors for these fields, which are not applicable to our study (we are analyzing aggregated data from independent resources, not collecting/reporting data from new biological specimens). We will need additional guidance on how the descriptions we provided may be modified to adequately complete this section, if we should remove our existing descriptions in favor of “not applicable” entries, or if another course of action for completing this form is appropriate. Currently consulting with @obigriffith and @malachig to investigate an inquiry to the editor before submitting full response along with the revised manuscript.

We evaluated our existing tables for the inclusion of additional descriptive text for legends, per the criteria set out in R_extended_comments. We believe that all tables meet these criteria, though we have reformatted per the guidelines indicated (bolding the table titles, and emulating double-spaced text for existing legends by doubling the legend cell height). A new legend was created for Supplementary table S8, to address concerns specific to that table regarding the calculation of p-values. We have also added the raw counts used in the statistic calculation as columns C-F of table S8.
